src/common/license/src/license.rs

```diff
@@ -82,25 +112,97 @@ pub struct LicenseInfo {
     pub r#type: Option<String>,
     pub org: Option<String>,
     pub tenants: Option<Vec<String>>,
-    pub features: Option<Vec<String>>,
+    pub features: Option<Vec<Feature>>,
```

Comment:
Will this break existing licenses?

Reply:
tested in https://github.com/datafuselabs/databend/pull/15336/files#diff-ecc00d87658b61b7d05f5b675362d6a71d462f76325bca55f4e5f14d95af74cfR144